tests/Aql/AqlValueMemoryLayoutTest.cpp

```diff
@@ -0,0 +1,419 @@
+  // ff ff
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2a, 0xff, 0xff, 0x00, UNINITIALIZED, UNINITIALIZED, UNINITIALIZED,
+    0xff, 0xff, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(65535ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_65536) {
+  // 01 00 00
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2a, 0x00, 0x00, 0x01, UNINITIALIZED, UNINITIALIZED, UNINITIALIZED,
+    0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(65536ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_65537) {
+  // 01 00 01
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2a, 0x01, 0x00, 0x01, UNINITIALIZED, UNINITIALIZED, UNINITIALIZED,
+    0x01, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(65537ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_1073741824) {
+  // 40 00 00 00
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2b, 0x00, 0x00, 0x00, 0x40, UNINITIALIZED, UNINITIALIZED,
+    0x00, 0x00, 0x00, 0x40, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(1073741824ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_197374285) {
+  // 75 a4 ec e9
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2b, 0xe9, 0xec, 0xa4, 0x75, UNINITIALIZED, UNINITIALIZED,
+    0xe9, 0xec, 0xa4, 0x75, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(1973742825ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_2147483647) {
+  // 7f ff ff ff
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2b, 0xff, 0xff, 0xff, 0x7f, UNINITIALIZED, UNINITIALIZED,
+    0xff, 0xff, 0xff, 0x7f, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(2147483647ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_2147483648) {
+  // 7f ff ff ff
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2c, 0x00, 0x00, 0x00, 0x80, 0x00, UNINITIALIZED,
+    0x00, 0x00, 0x00, 0x80, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(2147483648ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_4294967294) {
+  // ff ff ff fe
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2c, 0xfe, 0xff, 0xff, 0xff, 0x00, UNINITIALIZED,
+    0xfe, 0xff, 0xff, 0xff, 0x00, 0x00, 0x00, 0x00
+  };
+  runChecksForUInt64(uint64_t(4294967294ULL), expected);
+}
+
+TEST(AqlValueMemoryLayoutTest, UnsignedLargerValues48Bit_4294967295) {
+  // ff ff ff ff
+  uint8_t const expected[] = { 
+    AqlValue::AqlValueType::VPACK_INLINE_INT48, 0x2c, 0xff, 0xff, 0xff, 0xff, 0x00, UNINITIALIZED,
```

Comment:
Why is this value encoded with 5 byes? 4 would be enough..?

Reply:
As discussed, this is necessary because the value is encoded as a signed integer right now.